src/app/pages/dashboard/dashboard.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container-fluid my-5">
-  <p class="fs-5 pt-5 mb-5">Hello <span class="fw-bold">{{ userName }}</span>, this is an overview of your activity.</p>
+  <p class="fs-5 pt-5 mb-5">Hello_  <span class="fw-bold">{{ userName }}</span>, this is an overview of your activity.</p>
```

Comment:
Is this underscored intended?

Reply:
To make a space between the username and the hello, I tried adding " ", and didnt work.